functions/Find-DbaAgentJob.ps1

```diff
@@ -97,7 +97,7 @@ function Find-DbaAgentJob {
 			Returns all agent job(s) that have failed since July of 2016 (and still have history in msdb)
 
 		.EXAMPLE
-			Get-DbaRegisteredServerName -SqlInstance CMSServer -Group Production | Find-DbaAgentJob -Disabled -ExcludeSchedule -Detailed | Format-Table -AutoSize -Wrap
+			Get-DbaRegisteredServer -SqlInstance CMSServer -Group Production | Find-DbaAgentJob -Disabled -IsNotScheduled | Format-Table -AutoSize -Wrap
 
```

Comment:
hey cool, does [dbainstanceparameter] support the CMS server object? I forget.

Reply:
Yes, I added that in #2169 / f01d6d8a78e674930259005327da5700d250b508